app/models/piece.rb

```diff
@@ -94,7 +94,7 @@ def capture(to_x:, to_y:)
     return 'success' if target_piece.nil?
 
     # Valid move with enemy piece captured at destination
-    if !target_piece.nil? && !target_piece.pieces_turn?
+    if type != 'pawn' && !target_piece.nil? && !target_piece.pieces_turn?
```

Comment:
Great, this should ensure that a pawn can't make a head-on capture through the capture logic.